pkg/schema/in_memory.go

```diff
@@ -44,7 +44,7 @@ func (s *InMemoryService) Create(_ context.Context, name string, bytes []byte) (
 	return schema.Instance{
 		ID:      strconv.Itoa(created.ID),
 		Name:    created.Subject,
-		Version: created.Version,
+		Version: int32(created.Version),
```

Comment:
Btw, why do we use `int32` in `schema.Instance`? I anticipate that it will cause a bunch of casts in our code, also given that we'll expose this to the connector and processor this could simply be an `int` to make it easier to work with.

Reply:
I just noticed you changed it to `int` in https://github.com/ConduitIO/conduit-commons/pull/67 👍